src/Entity/User/User.php

```diff
@@ -70,7 +70,8 @@ class User implements AdvancedUserInterface, \Serializable, EquatableInterface
      *      min=8,
      *      max=4096,
      *      minMessage="Password must be at least {{ limit }} charactes long",
-     *      maxMessage="Password cannot be longer than {{ limit }} characters"
+     *      maxMessage="Password cannot be longer than {{ limit }} characters",
```

Comment:
These rules should probably also go into `AddUserCommand` (currently has a min of 6 chars).

Reply:
@adlearningmate I think you worked on this, can you please add the same rules to the `AddUserCommand`?